ckanext/keycloak/views.py

```diff
@@ -13,15 +12,16 @@
 
 keycloak = Blueprint('keycloak', __name__, url_prefix='/user')
 
-
 server_url = tk.config.get('ckanext.keycloak.server_url', environ.get('CKANEXT__KEYCLOAK__SERVER_URL'))
 client_id = tk.config.get('ckanext.keycloak.client_id', environ.get('CKANEXT__KEYCLOAK__CLIENT_ID'))
 realm_name = tk.config.get('ckanext.keycloak.realm_name', environ.get('CKANEXT__KEYCLOAK__REALM_NAME'))
 redirect_uri = tk.config.get('ckanext.keycloak.redirect_uri', environ.get('CKANEXT__KEYCLOAK__REDIRECT_URI'))
-client_secret_key = tk.config.get('ckanext.keycloak.client_secret_key', environ.get('CKANEXT__KEYCLOAK__CLIENT_SECRET_KEY'))
+client_secret_key = tk.config.get('ckanext.keycloak.client_secret_key',
+                                  environ.get('CKANEXT__KEYCLOAK__CLIENT_SECRET_KEY'))
```

Comment:
Is it not picking up the values from the .ini file?

Reply:
Hm, I'm not sure, may be it does